### PR TITLE
consent and waiver of hearing

### PR DESCRIPTION
**Pull request by ianialator**
Thanks for letting me give this a go, sorry it took so long to get it up. I still have a few TODO's in here that I'll either turn into issues or take care of as soon as I make it to a meeting.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (joeldenning) [inline .vscode/launch.json]: Please git ignore the `.vscode` directory. To do so:

1. Add `.vscode` to `.gitignore` file
2. `git rm --cached .vscode/launch.json`
3. `git add .`
4. `git commit`

File: src/app/forms/consent-and-waiver-of-hearing/consent-and-waiver-of-hearing.web.component.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <TextInput dataKey="case.judgeName" label={__("judge full name")} />
+      </Section>
+      <Section name="2. Form Specifics">
+        <Checkbox dataKey="" label="" options="" />
+        <Checkbox dataKey="" label="" options="" />
+        <Radio dataKey="" label="" options="" />
+        <Radio dataKey="" label="" options="" />
+        <Radio dataKey="" label="" options="" />
+      </Section>
+      {/*TODO: IS THIS FOR THE PROSECUTOR TOO?
```
Comment (joeldenning) [inline src/app/forms/consent-and-waiver-of-hearing/consent-and-waiver-of-hearing.web.component.js]: No, it is only going to be filled out by the petitioner. We don't need to fill in the prosecutor sections

File: src/app/forms/consent-and-waiver-of-hearing/consent-and-waiver-of-hearing.web.component.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        />
+        <TextInput
+          dataKey="case.addressCourtState"
+          label={__("court address state")}
+        />
+        <TextInput
+          dataKey="case.addressCourtZip"
+          label={__("court address zip")}
+        />
+        {/* TODO: WHERE IS "PETIONER"? <TextInput dataKey="" label={__("petioner")}/>*/}
```
Comment (joeldenning) [inline src/app/forms/consent-and-waiver-of-hearing/consent-and-waiver-of-hearing.web.component.js]: The petitioner field should be filled in just with the full name of the person filling out the form.

File: src/app/forms/consent-and-waiver-of-hearing/consent-and-waiver-of-hearing.pdf.component.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+          <PositionedCheckmark dataKey="" left="" top="" />
+
+          <PositionedCheckmark dataKey="" left="" top="" />
+        </RenderPage>
+
+        <RenderPage url="/static/forms/consent-waiver-of-hearing/consent_waiver_of_hearing-2.png">
+          <PositionedCheckmark dataKey="" left="" top="" />
+          <PositionedCheckmark dataKey="" left="" top="" />
+          {/* TODO: IS THIS FOR THE PROSECUTOR TOO? */}
```
Comment (joeldenning) [inline src/app/forms/consent-and-waiver-of-hearing/consent-and-waiver-of-hearing.pdf.component.js]: No - see other comment. This is only going to be used by the petitioner.

**Comment by seanbw89** [comment]
Closing PR seems to be abandoned.

**Comment by joeldenning** [comment]
Note that we're still using your code! But that we've created our own branch with it and have created a new PR in #87

**Pull request closed without merging**